docs/rules/no-array-reduce.md

```diff
@@ -39,10 +41,34 @@ Array.prototype.reduce.call(array, reducer);
 array.reduce(reducer, initialValue);
 ```
 
+```js
+array.reduce((total, value) => total + value;
```

Comment:
This is invalid code.

Reply:
Addressed in this [commit](https://github.com/sindresorhus/eslint-plugin-unicorn/pull/1418/commits/eb8b2260c5bf0fc0924d1ebce81ab50f1ce4e2bf).